backend/api/v1/endpoints/chatbot/huggingface.py

```
@@ -11,7 +11,7 @@
 from fastapi import APIRouter,UploadFile, File, Form, Depends, HTTPException
 from schemas.huggingface import PromptRequest, PromptResponse, BotResponse
 from core.dependencies import get_db
-from api.v1.endpoints.auth import get_current_user  # Authentication dependency
+from api.v1.endpoints.Auth.auth import get_current_user  # Authentication dependency
```

Ruff (0.8.2)

Module level import not at top of file

(E402)